### DISCUSSION TITLE
unable to update azuracast

### DISCUSSION THREAD

**Discussion by silvaalex2021** [AzuraCast/AzuraCast#4150] [Q&A] [answered]
hello,

first, i know i should have opened a new issue, but i couldn't do that.

I just don't know what's wrong. Everything seems to be working fine, but I can't update my azuracast installation.

When I run "./docker.sh update", there is no error message at all, all I have are messages saying that everything worked fine and that azuracast is now up to date, but this is not true and I am stuck on version 4789f62.

after a while, my Dashboard shows that there are new updates, and then I repeat the process, just so that everything happens again ...

Any ideas?

Thank you so much!

**Comment by BusterNeece** [comment, marked as answer]
@silvaalex2021 Hello! We ran into a very rare edge-case problem with our Docker image building process this weekend and have just been able to resolve it this evening. If you update again, everything should be back to normal working order. Sorry for the inconvenience.

**Reply by silvaalex2021** [reply]
@SlvrEagle23 thank you so much! everything works fine now!

You can imagine? A fully configured and optimized VPS being ruined as part of my efforts to see if I had broken anything? And guess what! No backups! 😰

definitely, my boss must be worshiping me! 😡😡
